### DISCUSSION TITLE
While installing dogecoind command, I get the error.

### DISCUSSION THREAD

**Discussion by Grin0502** [dogecoin/dogecoin#3654] [Help needed]
After installing dependencies, I followed these steps.

./autogen.sh
./configure
make
make install # optional

./autogen.sh and ./configure was succeed.
But when I execute "make" command, i got error.

![image](https://github.com/user-attachments/assets/bccee249-b74f-4b24-975a-23871a4f40d0)

Anyone can help me?

**Comment by chromatic** [comment]
Can you share which version of Boost/Boost-dev you have installed?
